haiku-driver.groovy

```diff
@@ -26,8 +31,77 @@ def updated() {
     log.debug "updated"
 }
 
+def setFanDirection(String direction) {
+    sendCommand("FAN", "DIR", "SET;${direction}")
+}
+
+def reverseFan() {
+    if (device.currentValue("fanDirection") == "FWD") {
+        setFanDirection("REV")
+    } else {
+        setFanDirection("FWD")
+    }
+}
+
+def refresh() {
```

Comment:
This looks unused - is it a Hubitat API? Does the GET result come through `parse` and update Hubitat with an event?

Reply:
Nevermind, missed the new capability above.